File: eTeatar/eTeatar.Model/StavkaUplate.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eTeatar.Model
{
    public class StavkaUplate
    {
        public int StavkaUplateId { get; set; }

        public int Kolicina { get; set; }

        public float Cijena { get; set; }

        public int HranaId { get; set; }

        public int UplataId { get; set; }

        public virtual Hrana Hrana { get; set; } 

        public virtual Uplata Uplata { get; set; }
    }
}

File: eTeatar/UI/eteatar_desktop/lib/models/stavka_uplate.dart
import 'package:json_annotation/json_annotation.dart';

part 'stavka_uplate.g.dart';

@JsonSerializable()
class StavkaUplate {
  int? stavkaUplateId;
  int? kolicina;
  double? cijena;
  int? hranaId;
  int? uplataId;

  StavkaUplate(
    this.stavkaUplateId,
    this.kolicina,
    this.cijena
  );

  factory StavkaUplate.fromJson(Map<String, dynamic> json) => _$StavkaUplateFromJson(json);

  /// Connect the generated [_$GlumacToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$StavkaUplateToJson(this);
}